docker/federation/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
 
   pictrs:
     restart: always
-    image: asonix/pictrs:0.3.1
+    image: asonix/pictrs:0.4.0-beta.19
```

Comment:
Hope this doesn't have any breaking changes. We gotta test this before prod deploys if this gets merged.